mapbox/libjava-services/src/main/java/com/mapbox/services/api/directions/v5/MapboxDirections.java

```diff
@@ -373,6 +375,17 @@ public T setLanguage(String language) {
       return (T) this;
     }
 
+    /**
+     * Optionally, call if you'd like to specify if exit on a roundabout is allowed.
```

Comment:
"Optionally, set this to true if you want to enable instructions while exiting roundabouts and rotaries."